src/averageAges.js

```diff
@@ -14,12 +14,21 @@
  *
  * @return {number}
  */
+function averageAge(people) {
+  return people.reduce(
+    (acc, curr) => acc + (curr.died - curr.born), 0
+  ) / people.length;
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(
+    century
+      ? person => person.sex === 'm'
+        && Math.ceil(person.died / 100) === century
+      : person => person.sex === 'm'
+  );
```

Comment:
```suggestion
  const men = people.filter(person => century
      ? person.sex === 'm'
        && Math.ceil(person.died / 100) === century
      : person.sex === 'm'
  );
```

Reply:
1 - you'll accept same argument, so there is no need in two callbacks
2 - you can return condition from array function and wrap cases to new lines